stdlib/public/core/Process.swift

```diff
@@ -56,10 +56,18 @@ public enum Process {
   /// around by the optimizer which will break the data dependence on argc
   /// and argv.
   public static var arguments: [String] {
-    if _arguments == nil { 
-      _arguments = _computeArguments()
+    let argumentsPtr = UnsafeMutablePointer<AnyObject?>(
+      Builtin.addressof(&_swift_stdlib_ProcessArguments))
```

Comment:
I know I'm a little late here, but can we do this without Builtin.addressof? Seems like plain old withUnsafeMutablePointer should be good enough. (I know it does the same thing underneath, but it's better to keep that kind of raw access in one place.)